command/bridge/deploy/internal_contracts.go

```diff
@@ -0,0 +1,367 @@
+//nolint:dupl
+package deploy
+
+import (
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/validator"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/Ethernal-Tech/ethgo"
+)
+
+var (
+	bigZero = big.NewInt(0)
+
+	internalContracts []*contract
```

Comment:
Delete this
```suggestion
	internalContracts []*contract
```

Reply:
https://github.com/Ethernal-Tech/blade/pull/372/commits/eeac49723811da651c1123e7c9370c7d3f58fc70